### PR TITLE
Don't throw error on empty command;fix init rocket

### PR DESCRIPTION
**Pull request by peterdresslar**

### CODE REVIEW

File: packages/commands/init/src/index.ts [hunk 1]
```diff
@@ -514,15 +514,15 @@ export const plugin: CliPlugin = {
...
-                    Next Steps:
-                    - Change directory into project folder - ${chalk.cyan(`cd ${projectPath}`)}
-                    - Install ${chalk.cyan(`yarn install`)} to install dependencies
-                    ${initializationType !== InitializationType.ExistingGraphQL ? `
-                    - Edit the .graphql file inside your model folder.
-                    - Run ${chalk.cyan(`yarn graphql generate`)} to generate schema and resolvers
-                    - Run ${chalk.cyan(`yarn graphql codegen`)} to generate TypeScript typings
-                    ` : ''}
+                    console.info(`🚀  GraphQL CLI project successfully initialized into the folder; ${projectPath}
+                  Next Steps:
```
Comment (wtrocki) [inline packages/commands/init/src/index.ts]: Actually this indentation can be removed as it makes reading entire log harder

**Comment by peterdresslar** [comment]
I thought about that but it will really make the code look bad. Could also
use a utility function but that's a bit bigger change?

**Comment by wtrocki** [comment]
This works for me, but leaving final merge decition to @ardatan

File: packages/cli/src/cli.ts [hunk 2]
```diff
@@ -1,16 +1,18 @@
...
+import chalk from 'chalk';
 
 export async function cli(argv = process.argv): Promise<void> {
   try {
     const rootCommand = argv[2];
 
     if (!rootCommand || rootCommand === '') {
-      throw new Error(`
-      You need to specify a valid command!
+      console.error(`Usage: ${chalk.cyan(`graphql`)} requires a command to run successfully:
```
Comment (ardatan) [inline packages/cli/src/cli.ts]: Why don't we throw the error? It is already caught by try/catch.

**Comment by peterdresslar** [comment]
Throwing an error with an empty command as it is now causes the Command
program to exit with a stack trace to the command line. We should be able
to just terminate the Program with a graceful shutdown message.

**Pull request merged**